apps/gitness/src/pages-v2/profile-settings/settings-layout.tsx

```diff
@@ -0,0 +1,17 @@
+import { Outlet } from 'react-router-dom'
+
+import { SettingsAccountPage } from '@harnessio/ui/views'
+
+import Breadcrumbs from '../../components/breadcrumbs/breadcrumbs'
+
+export const SettingsLayout = () => {
+  return (
+    <>
+      <div className="bg-background-1 sticky top-0 z-40">
+        <Breadcrumbs />
+        <SettingsAccountPage />
```

Comment:
why is the whole page inside the `sticky` div?

Reply:
Its the `breadcrumbs` and `tab-nav`, the tab nav is named `SettingsAccountPage`, will rename in a followup PR. The actual page is rendered through the outlet.